x-pack/plugins/infra/public/containers/map/with_map.ts

```diff
@@ -3,22 +3,25 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+import { get } from 'lodash';
 import moment from 'moment';
 import { graphql } from 'react-apollo';
+import uuid from 'uuid';
 import {
   InfraFilterInput,
   InfraMetricInput,
   InfraMetricType,
   InfraPathInput,
   InfraPathType,
-  InfraResponse,
   InfraSource,
   InfraTimerangeInput,
 } from '../../../common/graphql/types';
+import { InfraWaffleMapGroup } from '../../lib/lib';
+import { nodesToWaffleMap } from './nodes_to_wafflemap';
 import { query } from './query';
```

Comment:
The query imported here is named `Query`. That's a very conflict-prone name, especially since it will end up in the generated types. How about we call it something like

```typescript
export const mapQuery = gql`
  query MapQuery(...) {
    ...
  }
`;
```

Then the generated query namespace would be `MapQuery` in `common/graphql/types.ts` (with the query result type as `MapQuery.Query` and the variables as `MapQuery.Variables`, see below).